File: 2023/domaci/46-rozbalovac/volfmatous/main.py
import os
import sys
import zipfile

if __name__ == "__main__":
    file_path = input()

    if not os.path.exists(file_path):
        print("not found")
        sys.exit()

    with zipfile.ZipFile(file_path, "r") as zip_ref:
        with zip_ref.open("input.txt") as text_file:
            for line in text_file:
                print(line.decode("utf-8").strip())

File: 2023/domaci/46-rozbalovac/volfmatous/main.cs
using System;
using System.IO;
using System.IO.Compression;

internal class Program
{
    static void Main(string[] args)
    {
        string filePath = Console.ReadLine();

        if (!File.Exists(filePath))
        {
            Console.WriteLine("not found");
            return;
        }

        using (ZipArchive zipArchive = ZipFile.OpenRead(filePath))
        {
            foreach (ZipArchiveEntry entry in zipArchive.Entries)
            {
                if (entry.FullName == "input.txt")
                {
                    using (StreamReader reader = new StreamReader(entry.Open()))
                    {
                        string line;
                        while ((line = reader.ReadLine()) != null)
                        {
                            Console.WriteLine(line);
                        }
                    }
                    return;
                }
            }
        }
    }
}
